src/node.h

```diff
@@ -306,9 +306,17 @@ class NODE_EXTERN InitializationResult {
 // better suited for a public embedder API.
 NODE_EXTERN int Start(int argc, char* argv[]);
 
+enum StopFlags {
+  kNoFlags = 1 << 0,
+  // Do not explicitly terminate the Isolate
+  // when exiting the Environment.
+  kDoNotTerminateIsolate = 1 << 1,
```

Comment:
```suggestion
  kNoFlags = 0,
  // Do not explicitly terminate the Isolate
  // when exiting the Environment.
  kDoNotTerminateIsolate = 1 << 0,
```
Having 1 mean "no flags" feels kinda weird because then what does 0 signify?